src/dynamic-page.scss

```diff
@@ -272,6 +282,7 @@ $block: #{$fd-namespace}-dynamic-page;
 
   .#{$fd-namespace}-toolbar.#{$block}__title-content-toolbar {
```

Comment:
Can you remove the `.#{$fd-namespace}-toolbar` ? This is not making the component self-contained. If the rules in `.#{$block}__title-content-toolbar` are not taking precedence over the ones coming from ` .#{$fd-namespace}-toolbar` you should apply more specificity.

Reply:
done, also removed mixed scope from breadcrumb and tabs.